Source/Core/Core/HW/EXI/EXI_DeviceEthernet.cpp

```diff
@@ -139,7 +141,7 @@ void CEXIETHERNET::ImmWrite(u32 data, u32 size)
 
   // Reach here if we're actually writing data to the EXI or MX region.
 
-  DEBUG_LOG(SP1, "%s write %0*x", transfer.region == transfer.MX ? "mx " : "exi", size * 2, data);
+  DEBUG_LOG_FMT(SP1, "{} write {:x}", transfer.region == transfer.MX ? "mx " : "exi", data);
```

Comment:
Not sure why there was a 0 padding based on the size multiplied by 2. I'm fine with this being removed.